airflow/settings.py

```diff
@@ -352,7 +353,8 @@ def dispose_orm():
     global Session
 
     if Session is not None:
-        Session.remove()
+
+Session.remove()
```

Comment:
This looks odd? Was this change intentional?

Reply:
Bad merge. I knew I should not use GitHub UI for that. Lazy, Lazy Jarek.

Reply:
Thanks @o-nikolas  for vigilance. Resolved.